model/docs/pyrenew_demo.qmd

```diff
@@ -112,10 +119,12 @@ latent_hospitalizations = HospitalAdmissions(
 
 # 5) An observation process for the hospitalizations
 observed_hospitalizations = PoissonObservation()
+# Damon: What does it mean that there is a PoissonObservation? What are the parameters of the Poisson distrubtion?
```

Comment:
Good point, the `pyrenew_demo.qmd` needs to include a model description as initial section.